frontend/packages/ux-editor/src/components/Elements/LayoutSetsContainer.test.tsx

```diff
@@ -12,35 +12,73 @@ import {
 import { QueryKey } from 'app-shared/types/QueryKey';
 import { appContextMock } from '../../testing/appContextMock';
 import { app, org } from '@studio/testing/testids';
+import {
+  addFeatureFlagToLocalStorage,
+  removeFeatureFlagFromLocalStorage,
+} from 'app-shared/utils/featureToggleUtils';
+import { textMock } from '@studio/testing/mocks/i18nMock';
 
 // Test data
 const layoutSetName1 = layoutSet1NameMock;
 const layoutSetName2 = layoutSet2NameMock;
 
 describe('LayoutSetsContainer', () => {
   it('renders component', async () => {
+    const user = userEvent.setup();
     render();
 
+    const combobox = screen.getByRole('combobox');
+    await user.click(combobox);
+
```

Comment:
You mean the new line?

Reply:
No all the added lines? Since the test case is only "render the component"? But I guess it is because the native select is changed to a combobox. So maybe the test case should be adapted to something like: `renders the layout set names as options in the dropdown`?

Reply:
ah, sorry missed the title of the test. 🙈 Yes, for sure it should be a lot more descriptive. From my point of view, testing that a component is rendering is done indirectly throughout other tests that are testing the component. They would definitely fail if rendering fails.